jetstream/cli.py

```diff
@@ -56,28 +54,71 @@ class AllType:
 
 class ExecutorStrategy(Protocol):
     project_id: str
-    dataset_id: str
 
-    def __init__(self, project_id: str, dataset_id: str, *args, **kwargs) -> None:
+    def __init__(self, project_id: str, *args, **kwargs) -> None:
```

Comment:
I think we're overloading project_id a little here -- I thought these were about where the analysis tables should land, which is maybe different from the project that Argo should execute in. Do you think it matters? Should we be able to ask Argo to put tables somewhere else?

Reply:
You are right. I added support for configuring both also in the workflow yaml file